packages/react-html/src/manager.ts

```diff
@@ -123,4 +141,24 @@ export class HtmlManager {
       this.updateSubscriptions();
     }
   }
+
+  private removeHtmlAttributes(
+    attributes: React.HtmlHTMLAttributes<HTMLHtmlElement>,
+  ) {
+    const index = this.htmlAttributes.indexOf(attributes);
+
+    if (index >= 0) {
```

Comment:
We have a lot of functions that do more or less the same thing with different collections, took me a minute to understand what the differences were and why we did it this way, might it make sense to stick this into a `removeAttributes` helper?

Probably the same deal for adding attributes too I suppose.

Alternatively attribute lists could be a class that has an `add` and `remove` function of their own and have the `updateSubscriptions` callback given to them on initialization, but that might be a little overkill.

Reply:
DRY'ed up the methods that just did push to array + splice later on 👍